.husky/pre-push

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn prepush
+#  yarn prepush
```

Comment:
lol

Reply:
when you open a new PR you lose [context](https://github.com/finos/perspective/pull/1799/files#r867238799)